.github/workflows/stale.yml

```diff
@@ -0,0 +1,31 @@
+name: Stale PRs
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Label/close stale PRs
+        uses: actions/stale@v8
+        with:
+          stale-pr-message: 'This PR is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
+          close-pr-message: 'This PR was closed because it has been stalled for 10 days with no activity.'
+
+          # Stale after 60 days, close after further 10 days
+          days-before-pr-stale: 60
+          days-before-pr-close: 10
+
+          # Don't work on issues (only PRs)
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+
+          # Only consider 'community' PRs
+          only-labels: 'community'
```

Comment:
Do we really want to consider only community PRs? From my perspective we could include org members' PRs as well, so we also get a better feeling of how this automation "feels" if you are the author.